tests/test_url.py

```diff
@@ -1307,26 +1307,31 @@ def test_with_suffix_replace():
 def test_is_absolute_for_relative_url():
     url = URL("/path/to")
     assert not url.is_absolute()
+    assert not url.absolute
```

Comment:
@bdraco cached things are usually tested through their `__wrapped__` attributes to prevent caching from influencing behavior in tests. Have you considered doing something like this?

Reply:
Since the URL object is immutable and the state is already predefined we shouldn’t have much risk of the cached value being calculated incorrectly.

Was there a specific case you are concerned about?

Reply:
No, just got surprised seeing this and thought I'd mention it.